RSPlayPauseButtonDemo/ViewController.m

```diff
@@ -30,6 +30,7 @@ - (void)viewDidLoad
     
     if (!self.playPauseButton) {
         self.playPauseButton = [[RSPlayPauseButton alloc] init];
+        self.playPauseButton.hasBorderCircle = YES;
```

Comment:
Setting these properties should probably be reactive, so that if e.g. the color changes during runtime, the control updates.